### PR TITLE
feat(Core/Time): remove inherited ACE Time

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## CHANGES PROPOSED:
-  remove inherited ACE Time

## ISSUES ADDRESSED:
- Part #3451

## TESTS PERFORMED:
- Build, test in-game, test in the live server (1+ month)
- https://youtu.be/fAy1M_NLEGg

## HOW TO TEST THE CHANGES:
- I don't know, something related to time.

## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Winfidonarleyan** [comment]
@Viste need review

**Review by Viste — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
Added video with test this PR

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan when it's the time tested in the video? which action depends on time?

**Comment by Winfidonarleyan** [comment]
I made a test in general, I don't know how else to check the time, but if it didn't work, I would have seen.

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan @Viste ok, looks good to me. Thanks!

File: src/common/Utilities/Util.cpp [hunk 1]
```diff
@@ -102,6 +102,45 @@ Tokenizer::Tokenizer(const std::string &src, const char sep, uint32 vectorReserv
     }
 }
 
+#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__))
+struct tm* localtime_r(time_t const* time, struct tm* result)
+{
+    localtime_s(result, time);
+    return result;
+}
+#endif
```
Comment (Helias) [inline src/common/Utilities/Util.cpp:111]: actually, we already have this, but it's not ready merged to master
https://github.com/azerothcore/azerothcore-wotlk/pull/3365/files

Comment (Viste) [reply src/common/Utilities/Util.cpp:111]: @Helias this part in fact must be in this PR and #3365 must be next after this

**Pull request merged**